test/system/full_text_search/search_test.rb

```diff
@@ -60,8 +60,14 @@ def test_keep_search_target
       click_on("search-target-wiki-pages")
       fill_in("search-input", with: "cookbook")
       click_on("search-submit")
-      assert_equal("selected",
-                   find(:link, "search-target-wiki-pages")["class"])
+      within("#search-result") do
+        within("#search-result-content") do
+          within("#search-source-types") do
+            assert find_link("search-target-wiki-pages",
+                             class: "selected")
```

Comment:
Do we need this `assert`?

`find_link` raises an exception on not found, right?

FYI: https://www.clear-code.com/blog/2012/1/18.html